xml/security_fips.xml

```diff
@@ -30,11 +30,10 @@
     </dm:docmanager>
   </info>
   <important>
-    <title>&productname; &productnumber; and FIPS 140-3</title>
+    <title>&productname; 15 SP4 and FIPS 140-3</title>
```

Comment:
If this needs to point to SP4, please use phrase here as this part is common for SLE Micro.

Reply:
Good point, I will add profiling.